swhkd/src/environ.rs

```diff
@@ -0,0 +1,105 @@
+use std::{env::VarError, path::PathBuf};
+
+pub struct Env {
+    pub pkexec_id: u32,
+    pub xdg_config_home: PathBuf,
+    pub xdg_runtime_socket: PathBuf,
+    pub xdg_runtime_dir: PathBuf,
+}
+
+#[derive(Debug)]
+pub enum EnvError {
+    PkexecNotFound,
+    XdgConfigNotFound,
+    XdgRuntimeNotFound,
+    GenericError(String),
+}
+
+impl Env {
+    pub fn contruct() -> Self {
```

Comment:
Is this misspelled?

Reply:
Haha 😄 

Yes it is misspelled I'm gonna fix it ASAP....LSP never made me realize it was :)